walkthroughs/tls-with-acm/src/colorteller/go.mod

```diff
@@ -2,10 +2,23 @@ module github.com/aws/aws-app-mesh-examples/walkthroughs/tls-with-acm/src/colort
 
 go 1.17
 
+require github.com/aws/aws-xray-sdk-go v1.6.0
+
 require (
 	github.com/DATA-DOG/go-sqlmock v1.5.0 // indirect
-	github.com/aws/aws-sdk-go v1.19.0 // indirect
-	github.com/aws/aws-xray-sdk-go v1.6.0
-	github.com/stretchr/testify v1.7.0 // indirect
-	golang.org/x/net v0.0.0-20210929193557-e81a3d93ecf6 // indirect
+	github.com/andybalholm/brotli v1.0.1 // indirect
+	github.com/aws/aws-sdk-go v1.44.207 // indirect
+	github.com/golang/protobuf v1.4.3 // indirect
+	github.com/jmespath/go-jmespath v0.4.0 // indirect
+	github.com/klauspost/compress v1.11.8 // indirect
+	github.com/pkg/errors v0.9.1 // indirect
+	github.com/stretchr/testify v1.8.1 // indirect
+	github.com/valyala/bytebufferpool v1.0.0 // indirect
+	github.com/valyala/fasthttp v1.24.0 // indirect
+	golang.org/x/net v0.7.0 // indirect
+	golang.org/x/sys v0.5.0 // indirect
+	golang.org/x/text v0.7.0 // indirect
+	google.golang.org/genproto v0.0.0-20210114201628-6edceaf6022f // indirect
+	google.golang.org/grpc v1.35.0 // indirect
+	google.golang.org/protobuf v1.25.0 // indirect
```

Comment:
Any context for why adding so many new mods?

Reply:
These values were added by `go mod tidy`, which is a standard good practice to normalize the dependency structure / resolutions.

This just hadn't been run here before, so it's causing some change that aren't strictly related to the updated client.

Reply:
Good to know, thanks